### ISSUE TITLE
Error: Cannot find module 'rsvp', when attempting firebase (v3.18.3) deploy on Travis

### ISSUE THREAD

**Issue by chalin** [firebase/firebase-tools#746]
### Version info

firebase-tools **v3.18.3**

### Platform Information

Linux (Travis)

### Steps to reproduce

https://github.com/dart-lang/site-www (and the sibling repo site-webdev) are [failing to deploy on Travis](https://travis-ci.org/dart-lang/site-www/jobs/365876476) since v3.18.3. 

### Expected behavior

Deployment from Travis should succeed. It did when we were using **v3.18.2** (as shown by the [job](https://travis-ci.org/dart-lang/site-www/jobs/365465348) previous to the one that is currently failing).

### Actual behavior

This is the reported error:

```nocode
module.js:549
    throw err;
    ^
Error: Cannot find module 'rsvp'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/.nvm/versions/node/v8.11.1/lib/node_modules/firebase-tools/lib/auth.js:11:12)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

cc @kwalrath @kevmoo

**Comment by mbleigh** [comment]
I'll take a look later this evening

**Comment by mbleigh** [comment]
Fixed in #747 and a new patch release will be cut tomorrow. Looks like a bad merge in a refactor we recently did caused the issue, but it didn't crop up unless you were doing a fresh install of `firebase-tools` from scratch.

**Comment by mbleigh** [comment]
Should be fixed in 3.18.4
